src/utils/iconutils.cpp

```diff
@@ -46,7 +48,9 @@ QIcon IconUtils::fetchIcon(const QString &p_iconFile,
 QIcon IconUtils::fetchIcon(const QString &p_iconFile, const QString &p_overriddenForeground)
 {
     QVector<OverriddenColor> colors;
-    if (!p_overriddenForeground.isEmpty()) {
+    const auto &themeMgr = VNoteX::getInst().getThemeMgr();
+
```

Comment:
As discussed, let's add a key-value config in the theme palette file to indicates whether the icon is monocolor.